routes/web.php

```diff
@@ -146,6 +146,7 @@ function () {
                 Route::delete("{discount_group}/filter/{product_filter}", ["as" => "product-filter.detach", "uses" => "DiscountGroupController@detachProductFilter"]);
             }
         );
+        Route::delete("/admin/discount-group/{discount_group}", ["as" => "admin.discount-group.softDelete", "uses" => "DiscountGroupController@softDelete"]);
```

Comment:
Also you have to do the format change here.

admin.discount-group.softDelete -> admin.discount-group.soft-delete